low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -151,7 +151,7 @@ def initialize(self, conf_reader, msgQlist, product):
         self._raid_device = "N/A"
         self.os_sensor_type = {
             "disk_space" : self.disk_sensor_data,
-            "system" : self.host_sensor_data,
+            "memory_usage" : self.host_sensor_data,

```

Comment:
@shriya-deshmukh  @Sumedh-a-k to fix test failure, we changed code in sspl? Can you remind me if CSM is depending on memory_usage or system ? If memory_usage, without this change then, CSM wont be getting these alerts then?

Reply:
The messages we send after a fixed interval of time (30s) already has resource-type as 'memory_usage' and @shriya-deshmukh checked and ensured that CSM UI was showing the `memory_usage` alerts for the same invocation, in sspl-test as well we are passing 'memroy_usage' as resource type just in the 'if condition in this file' we used `system` instead of 'memeory_usage' that was causing the issue.
@cdeshmukh